components/speedreader/rust/lib/src/readability/examples/slideshow.rs

```diff
@@ -0,0 +1,165 @@
+extern crate readability;
```

Comment:
Note: this file is for end-to-end testing only. we create a slideshow of before and after reader view renders. Since reader mode is how the page feels, this is how we can see big picture changes. We test individual components of the algorithm in doc tests.